src/models/disassemblyoutput.cpp

```diff
@@ -199,8 +200,24 @@ ObjectdumpOutput objdumpParse(const QByteArray& output)
             assembly = asmLine;
         }
 
-        disassemblyLines.push_back(
-            {addr, assembly, extractLinkedFunction(asmLine), {currentSourceFileName, sourceCodeLine}});
+        int branchVisualisationRange = 0;
+        for (const auto& character : std::as_const(assembly)) {
+            // format is the following:
+            //    /-  a5 54 12 ...
+            //    |   64 a3 ....
+            //    \-> 65 23 ....
+            // so we can simply skip all characters until we meet a letter or a number
+            if (character.isNumber() || character.isLetter()) {
```

Comment:
I think this should be either `if (character.isNumber() || character.isLower()) {` or, possibly change to a lookup string as this won't use the locale under the hood and the Qt doc for `isNumber()` explicit warns that it returns true for
>[Number_](https://doc.qt.io/qt-5/stylesheet-reference.html#number)* categories, not just 0-9

That would be possibly something like

```c
static QString const hex("0123456789abcdef");

        for (const auto& character : std::as_const(assembly)) {
            // format is the following:
            //    /-  a5 54 12 ...
            //    |   64 a3 ....
            //    \-> 65 23 ....
            // so we can simply skip all characters until we meet a letter or a number
            if (hex.indexOf(character) != -1) {
                break;
            }
```